tests/foreman/api/test_syncplan.py

```diff
@@ -295,10 +319,21 @@ def test_remove_one_product_to_sync_plan(self):
         @Feature: Sync Plan
 
         """
+        syncplan_id = entities.SyncPlan(
+            organization=self.org_id
+        ).create_json()['id']
+        product_id = entities.Product(
```

Comment:
The docstring says ``Create a Sync Plan with two products...`` :) you test is valid but I am interested in seeing this specific scenario where you end up leaving one repo behind.